contracts/HubPool.sol

```diff
@@ -225,7 +242,7 @@ contract HubPool is Testable, Lockable, MultiCaller, Ownable {
         uint64 poolRebalanceLeafCount,
         bytes32 poolRebalanceRoot,
         bytes32 destinationDistributionRoot
-    ) public onlyIfNoActiveRequest {
+    ) public noActiveRequests {
         require(poolRebalanceLeafCount > 0, "Bundle must have at least 1 leaf");
```

Comment:
Is there a way to verify that `poolRebalanceRoot` actually contains `poolRebalanceLeafCount` leafs?

Reply:
I dont think you can do this by just looking at the root 🤔 . you will need to know the data that went into constructing the leafs. if this is wrong the whole bundle is disputable.